### PR TITLE
Remove lein dotfiles

### PR DESCRIPTION
**Pull request by marco-m**

### CODE REVIEW

**Comment by marco-m** [comment]
Please apply first this PR and then the one that adds travis (but at that point it should be rebased before)

File: example/.lein-classpath [hunk 1]
```diff
@@ -1 +0,0 @@
-../src
```
Comment (RickMoynihan) [inline example/.lein-classpath]: Did you intend to remove this file?

This was added explicitly so you can dev/test the plugin without having to `lein install` it everytime.  [See here](https://github.com/technomancy/leiningen/blob/stable/doc/PLUGINS.md#local-development).  

Perhaps its presence means it's less of an `example` project though, and more a project for development/testing.

What do you think?

Comment (marco-m) [reply example/.lein-classpath]: Ah I understand now! And, as you mention, this is not an `example` project but a test project in my opinion. I have to think a bit more and propose something that makes sense. I will update this PR to remove only the `.lein-repl-history` file

Comment (RickMoynihan) [reply example/.lein-classpath]: Yeah.  My intention was for it to serve as both, rather than having to maintain two.

**Comment by marco-m** [comment]
I updated the PR accordingly to the discussion (do not remove `example/.lein-classpath` for the time being)

**Pull request closed without merging**